paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -160,8 +161,12 @@ def paasta_start_or_stop(args, desired_state):
     """Requests a change of state to start or stop given branches of a service."""
     soa_dir = args.soa_dir
     service = figure_out_service_name(args=args, soa_dir=soa_dir)
-    instances = args.instances.split(",") if args.instances else None
 
+    pargs = paasta_args_mixer(args, service)
```

Comment:
Indeed.
I added the options and updated tests to pass a real argparse tuple instead of mocking it.

This reveals that --cluster is a mandatory option for paasta start/stop/restart. That doesn't make lots of sense in combination with --deploy-group.
For now I left it as it was. But we might want to reconsider the way it works.